app/cdap/components/Replicator/Create/Content/SelectTables/index.tsx

```diff
@@ -450,6 +450,12 @@ class SelectTablesView extends React.PureComponent<ISelectTablesProps, ISelectTa
         <div className="text-danger">
           <Heading type={HeadingTypes.h5} label="Error" />
           <span>{this.state.error}</span>
+          <div>
```

Comment:
is this the only type of error that can be returned? I think you need to check if the database name is incorrect and conditionally add more error message

Reply:
You're right, will update accordingly